<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>placeholder</class>
 <widget class="QWidget" name="placeholder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Clock Component</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="placeholderLabel">
       <property name="text">
        <string>Format component:</string>
       </property>
       <property name="buddy">
        <cstring>placeholderComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KComboBox" name="placeholderComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="shortFormLabel">
       <property name="text">
        <string>Short form</string>
       </property>
       <property name="buddy">
        <cstring>shortFormCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="shortFormCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="leadingZerosLabel">
       <property name="text">
        <string>Show leading zeros</string>
       </property>
       <property name="buddy">
        <cstring>leadingZerosCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="leadingZerosCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="hoursModeLabel">
       <property name="text">
        <string>12 hours mode</string>
       </property>
       <property name="buddy">
        <cstring>hoursModeCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="hoursModeCheckBox">
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="textualFormLabel">
       <property name="text">
        <string>Textual form</string>
       </property>
       <property name="buddy">
        <cstring>textualFormCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="textualFormCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="possessiveFormLabel">
       <property name="text">
        <string>Possessive form</string>
       </property>
       <property name="buddy">
        <cstring>possessiveFormCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="possessiveFormCheckBox">
       <property name="text">
        <string/>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="previewLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
